api/src/main/java/org/openmrs/module/appointments/dao/AppointmentDao.java

```diff
@@ -12,6 +12,8 @@
 
 public interface AppointmentDao {
     List<Appointment> getAllAppointments(Date forDate);
+    List<Appointment> getAllAppointments(Date forDate, AppointmentStatus status);
```

Comment:
Add a new line between these, just so that this is consistent with the rest of the file